tests/diagnostics/uninitialized-globals.slang

```diff
@@ -45,4 +45,34 @@ float use_later_writte()
     return writtenLater;
 }
 
+// Should not warn here (glsl layout decoration check)
+layout(location = 0) in vec4 data;
+
+vec4 glsl_layout_in_ok()
+{
+    return data;
+}
+
+// Invalid use of layout variables
+layout(location = 1) out vec4 x;
+
+vec4 glsl_layout_out_undefined()
+{
+    //CHK-DAG: ([[# @LINE + 1]]): warning 41017: use of uninitialized global variable 'x'
+    return x;
+}
+
+// OK use of layout variables
```

Comment:
I don't think this comment is needed if we explain why we warn (programmer should see that the variable is initialized).